### PR TITLE
Fix regression in web ui for retention

### LINKED ISSUE #10283: [resolved] bucket default retention not working from UI
__Minio Version__: RELEASE.2020-08-16T18-39-38Z
```
./mc mb testminioclu/test --with-lock

aws --no-verify-ssl --endpoint-url http://172.27.0.3:9000 s3api put-object-lock-configuration --bucket test --object-lock-configuration 'ObjectLockEnabled="Enabled",Rule={DefaultRetention={Mode="COMPLIANCE",Days=1}}'

aws --no-verify-ssl --endpoint-url http://172.27.0.3:9000 s3api get-bucket-versioning --bucket test
{
    "Status": "Enabled"
}

aws --no-verify-ssl --endpoint-url http://172.27.0.3:9000 s3api get-object-lock-configuration --bucket test
{
    "ObjectLockConfiguration": {
        "ObjectLockEnabled": "Enabled",
        "Rule": {
            "DefaultRetention": {
                "Mode": "COMPLIANCE",
                "Days": 1
            }
        }
    }
}

./mc cp test1.txt testminioclu/test

aws --no-verify-ssl --endpoint-url http://172.27.0.3:9000 s3api get-object-retention --bucket test --key test1.txt
{
    "Retention": {
        "Mode": "COMPLIANCE",
        "RetainUntilDate": "2020-08-19T13:23:52+00:00"
    }
}


<upload test2.txt from UI>

aws --no-verify-ssl --endpoint-url http://172.27.0.3:9000 s3api get-object-retention --bucket test --key test2.txt

```

## Expected Behavior
```
{
    "Retention": {
        "Mode": "COMPLIANCE",
        "RetainUntilDate": "2020-08-19T13:23:52+00:00"
    }
}
```

## Current Behavior
```
An error occurred (NoSuchObjectLockConfiguration) when calling the GetObjectRetention operation: The specified object does not have a ObjectLock configuration
```

### PR DESCRIPTION
**Pull request by poornas**
Fixes: #10283 regression from PR #9259

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Documentation needed
- [ ] Unit tests needed

### CODE REVIEW

File: cmd/web-handlers.go [hunk 1]
```diff
@@ -1130,30 +1130,25 @@ func (web *webAPIHandlers) Upload(w http.ResponseWriter, r *http.Request) {
...
-			writeErrorResponse(ctx, w, errorCodes.ToAPIErr(s3Err), r.URL, guessIsBrowserReq(r))
-			return
-		}
+	// enforce object retention rules
+	retentionMode, retentionDate, legalHold, s3Err := checkPutObjectLockAllowed(ctx, r, bucket, object, getObjectInfo, retPerms, holdPerms)
+	if s3Err == ErrNone && retentionMode != "" {
+		opts.UserDefined[xhttp.AmzObjectLockMode] = string(retentionMode)
+		opts.UserDefined[xhttp.AmzObjectLockRetainUntilDate] = retentionDate.UTC().Format(iso8601TimeFormat)
+	}
+	if s3Err == ErrNone && legalHold.Status != "" {
```
Comment (kannappanr) [inline cmd/web-handlers.go]: you will never have legalhold set in webhandler

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :x: [more...](#827899b0-4114-4eed-941a-7f9512ae9049) |
| mint-gateway-azure.sh | :x: [more...](#d1bfa4f6-c4cd-437f-84ea-7ff20cb8f934) |
| mint-zoned.sh | :x: [more...](#0114f89d-11be-4f1e-a6aa-3e46bd990960) |
<a name="827899b0-4114-4eed-941a-7f9512ae9049"></a>
### 10285-8e62062/mint-gateway-s3.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev8.minio.io:31703
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp e74f61957fce:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 0 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 41 seconds
(4/15) Running aws-sdk-ruby tests ... done in 2 seconds
(5/15) Running awscli tests ... done in 1 minutes and 58 seconds
(6/15) Running healthcheck tests ... done in 0 seconds
(7/15) Running mc tests ... done in 36 seconds
(8/15) Running minio-dotnet tests ... done in 36 seconds
(9/15) Running minio-go tests ... FAILED in 1 minutes and 10 seconds
{
  "args": {
    "destination": {
      "Bucket": "minio-go-test-kjwikfra0et05abq",
      "Object": "srcObject",
      "Encryption": {},
      "UserMetadata": null,
      "ReplaceMetadata": false,
      "UserTags": null,
      "ReplaceTags": false,
      "LegalHold": "",
      "Mode": "",
      "RetainUntilDate": "0001-01-01T00:00:00Z",
      "Size": 0,
      "Progress": null
    },
    "source": {
      "Bucket": "minio-go-test-kjwikfra0et05abq",
      "Object": "srcObject",
      "VersionID": "",
      "MatchETag": "",
      "NoMatchETag": "",
      "MatchModifiedSince": "0001-01-01T00:00:00Z",
      "MatchUnmodifiedSince": "0001-01-01T00:00:00Z",
      "MatchRange": false,
      "Start": 0,
      "End": 0,
      "Encryption": null
    }
  },
  "duration": 311,
  "error": "This copy request is illegal because it is trying to copy an object to itself without changing the object's metadata, storage class, website redirect location or encryption attributes.",
  "function": "CopyObject(destination, source)",
  "message": "CopyObject failed",
  "name": "minio-go: testEncryptedSSES3ToSSES3CopyObject",
  "status": "FAIL"
}
(9/15) Running minio-java tests ... done in 19 seconds
(10/15) Running minio-js tests ... done in 43 seconds
(11/15) Running minio-py tests ... done in 1 minutes and 21 seconds
(12/15) Running s3cmd tests ... done in 59 seconds
(13/15) Running s3select tests ... done in 2 seconds
(14/15) Running security tests ... done in 0 seconds

Executed 14 out of 15 tests successfully.
```
<a name="0114f89d-11be-4f1e-a6aa-3e46bd990960"></a>
### 10285-8e62062/mint-zoned.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-c3.minio.io:31594
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp cb81dda20f18:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 2 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 44 seconds
(4/15) Running aws-sdk-ruby tests ... done in 5 seconds
(5/15) Running awscli tests ... done in 2 minutes and 16 seconds
(6/15) Running healthcheck tests ... done in 0 seconds
(7/15) Running mc tests ... done in 1 minutes and 22 seconds
(8/15) Running minio-dotnet tests ... done in 48 seconds
(9/15) Running minio-go tests ... done in 2 minutes and 30 seconds
(10/15) Running minio-java tests ... done in 1 minutes and 25 seconds
(11/15) Running minio-js tests ... done in 58 seconds
(12/15) Running minio-py tests ... done in 3 minutes and 16 seconds
(13/15) Running s3cmd tests ... FAILED in 20 seconds
{
  "name": "s3cmd",
  "duration": "8643",
  "function": "test_sync_list_objects",
  "status": "FAIL",
  "error": "WARNING: Bucket is not empty. Removing all the objects from it first. This may take some time...\nERROR: S3 error: 404 (Not Found)"
}
(13/15) Running s3select tests ... done in 8 seconds
(14/15) Running security tests ... done in 0 seconds

Executed 14 out of 15 tests successfully.
```
<a name="d1bfa4f6-c4cd-437f-84ea-7ff20cb8f934"></a>
### 10285-8e62062/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-c3.minio.io:31102
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp e6f4b00a8d1a:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 8 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 4 minutes and 7 seconds
(4/15) Running aws-sdk-ruby tests ... done in 18 seconds
(5/15) Running awscli tests ... done in 2 minutes and 55 seconds
(6/15) Running healthcheck tests ... done in 0 seconds
(7/15) Running mc tests ... done in 3 minutes and 1 seconds
(8/15) Running minio-dotnet tests ... done in 1 minutes and 30 seconds
(9/15) Running minio-go tests ... FAILED in 5 minutes and 38 seconds
{
  "args": {},
  "duration": 989,
  "error": "The specified multipart upload does not exist. The upload ID may be invalid, or the upload may have been aborted or completed.",
  "function": "CopyObjectPart(destination, source)",
  "message": "CompleteMultipartUpload call failed",
  "name": "minio-go: testUnencryptedToSSES3CopyObjectPart",
  "status": "FAIL"
}
(9/15) Running minio-java tests ... done in 6 minutes and 0 seconds
(10/15) Running minio-js tests ... done in 2 minutes and 35 seconds
(11/15) Running minio-py tests ... FAILED in 1 seconds
{
  "name": "minio-py:test_negative_make_bucket_invalid_name",
  "status": "FAIL",
  "args": {
    "location": "default value ('us-east-1')",
    "bucket_name": ".minio-py-test-3397cad7-3e21-4ca9-8779-e71ef2966f0c"
  },
  "message": "NoSuchBucket: message: The specified bucket does not exist.",
  "error": "Traceback (most recent call last):\n  File \"/mint/run/core/minio-py/tests.py\", line 147, in _call_test\n    func(log_entry, *args, **kwargs)\n  File \"/mint/run/core/minio-py/tests.py\", line 241, in test_negative_make_bucket_invalid_name\n    _call(log_entry, _CLIENT.make_bucket, bucket_name)\n  File \"/mint/run/core/minio-py/tests.py\", line 130, in _call\n    return func(*args, **kwargs)\n  File \"/usr/local/lib/python3.6/dist-packages/minio/api.py\", line 335, in make_bucket\n    raise ResponseError(response, method, bucket_name).get_exception()\nminio.error.NoSuchBucket: NoSuchBucket: message: The specified bucket does not exist.\n",
  "function": "make_bucket(bucket_name, location, object_lock)",
  "duration": 408
}
(11/15) Running s3cmd tests ... done in 1 minutes and 33 seconds
(12/15) Running s3select tests ... done in 1 minutes and 2 seconds
(13/15) Running security tests ... done in 0 seconds

Executed 13 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Review by kannappanr — APPROVED** [review]
LGTM

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-zoned.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :x: [more...](#ce8a738b-f11f-4b7d-93fd-d45cc1a57814) |
| mint-gateway-azure.sh | :x: [more...](#0944990b-1528-44a2-a0f5-cbf62155d764) |
<a name="ce8a738b-f11f-4b7d-93fd-d45cc1a57814"></a>
### 10285-7c56352/mint-gateway-s3.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev8.minio.io:30794
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 9d4b8215233a:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 1 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 41 seconds
(4/15) Running aws-sdk-ruby tests ... done in 2 seconds
(5/15) Running awscli tests ... done in 1 minutes and 7 seconds
(6/15) Running healthcheck tests ... done in 1 seconds
(7/15) Running mc tests ... done in 34 seconds
(8/15) Running minio-dotnet tests ... done in 29 seconds
(9/15) Running minio-go tests ... FAILED in 1 minutes and 44 seconds
{
  "args": {
    "destination": {
      "Bucket": "minio-go-test-1ovvg0ieunaepakq",
      "Object": "srcObject",
      "Encryption": {},
      "UserMetadata": null,
      "ReplaceMetadata": false,
      "UserTags": null,
      "ReplaceTags": false,
      "LegalHold": "",
      "Mode": "",
      "RetainUntilDate": "0001-01-01T00:00:00Z",
      "Size": 0,
      "Progress": null
    },
    "source": {
      "Bucket": "minio-go-test-1ovvg0ieunaepakq",
      "Object": "srcObject",
      "VersionID": "",
      "MatchETag": "",
      "NoMatchETag": "",
      "MatchModifiedSince": "0001-01-01T00:00:00Z",
      "MatchUnmodifiedSince": "0001-01-01T00:00:00Z",
      "MatchRange": false,
      "Start": 0,
      "End": 0,
      "Encryption": null
    }
  },
  "duration": 569,
  "error": "This copy request is illegal because it is trying to copy an object to itself without changing the object's metadata, storage class, website redirect location or encryption attributes.",
  "function": "CopyObject(destination, source)",
  "message": "CopyObject failed",
  "name": "minio-go: testEncryptedSSES3ToSSES3CopyObject",
  "status": "FAIL"
}
(9/15) Running minio-java tests ... done in 24 seconds
(10/15) Running minio-js tests ... done in 33 seconds
(11/15) Running minio-py tests ... done in 1 minutes and 37 seconds
(12/15) Running s3cmd tests ... done in 1 minutes and 0 seconds
(13/15) Running s3select tests ... done in 3 seconds
(14/15) Running security tests ... done in 0 seconds

Executed 14 out of 15 tests successfully.
```
<a name="0944990b-1528-44a2-a0f5-cbf62155d764"></a>
### 10285-7c56352/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-c3.minio.io:32098
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 8b3c0ad18a37:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 8 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 4 minutes and 26 seconds
(4/15) Running aws-sdk-ruby tests ... done in 17 seconds
(5/15) Running awscli tests ... done in 2 minutes and 42 seconds
(6/15) Running healthcheck tests ... done in 1 seconds
(7/15) Running mc tests ... done in 3 minutes and 21 seconds
(8/15) Running minio-dotnet tests ... done in 1 minutes and 32 seconds
(9/15) Running minio-go tests ... FAILED in 5 minutes and 19 seconds
{
  "args": {},
  "duration": 952,
  "error": "The specified multipart upload does not exist. The upload ID may be invalid, or the upload may have been aborted or completed.",
  "function": "CopyObjectPart(destination, source)",
  "message": "CompleteMultipartUpload call failed",
  "name": "minio-go: testUnencryptedToSSES3CopyObjectPart",
  "status": "FAIL"
}
(9/15) Running minio-java tests ... done in 5 minutes and 59 seconds
(10/15) Running minio-js tests ... done in 2 minutes and 43 seconds
(11/15) Running minio-py tests ... FAILED in 1 seconds
{
  "name": "minio-py:test_negative_make_bucket_invalid_name",
  "status": "FAIL",
  "args": {
    "location": "default value ('us-east-1')",
    "bucket_name": ".minio-py-test-1b19138c-800e-4515-a780-e3dabdf6d4ce"
  },
  "message": "NoSuchBucket: message: The specified bucket does not exist.",
  "error": "Traceback (most recent call last):\n  File \"/mint/run/core/minio-py/tests.py\", line 147, in _call_test\n    func(log_entry, *args, **kwargs)\n  File \"/mint/run/core/minio-py/tests.py\", line 241, in test_negative_make_bucket_invalid_name\n    _call(log_entry, _CLIENT.make_bucket, bucket_name)\n  File \"/mint/run/core/minio-py/tests.py\", line 130, in _call\n    return func(*args, **kwargs)\n  File \"/usr/local/lib/python3.6/dist-packages/minio/api.py\", line 335, in make_bucket\n    raise ResponseError(response, method, bucket_name).get_exception()\nminio.error.NoSuchBucket: NoSuchBucket: message: The specified bucket does not exist.\n",
  "function": "make_bucket(bucket_name, location, object_lock)",
  "duration": 415
}
(11/15) Running s3cmd tests ... done in 1 minutes and 27 seconds
(12/15) Running s3select tests ... done in 1 minutes and 2 seconds
(13/15) Running security tests ... done in 0 seconds

Executed 13 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Review by harshavardhana — APPROVED** [review]
LGTM & tested

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :x: [more...](#f60deaa7-37d3-4a32-a54d-3fb271dc547f) |
| mint-gateway-azure.sh | :x: [more...](#bd623953-953e-4533-817a-00ad21bf7ffc) |
| mint-zoned.sh | :x: [more...](#a960263d-1b36-4e47-bbf3-43022a790173) |
<a name="f60deaa7-37d3-4a32-a54d-3fb271dc547f"></a>
### 10285-820793f/mint-gateway-s3.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev6.minio.io:32012
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 4a0f3095bf19:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 0 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 41 seconds
(4/15) Running aws-sdk-ruby tests ... done in 2 seconds
(5/15) Running awscli tests ... done in 2 minutes and 1 seconds
(6/15) Running healthcheck tests ... done in 0 seconds
(7/15) Running mc tests ... done in 35 seconds
(8/15) Running minio-dotnet tests ... done in 30 seconds
(9/15) Running minio-go tests ... FAILED in 1 minutes and 5 seconds
{
  "args": {
    "destination": {
      "Bucket": "minio-go-test-91gscyzoxhpj2lv1",
      "Object": "srcObject",
      "Encryption": {},
      "UserMetadata": null,
      "ReplaceMetadata": false,
      "UserTags": null,
      "ReplaceTags": false,
      "LegalHold": "",
      "Mode": "",
      "RetainUntilDate": "0001-01-01T00:00:00Z",
      "Size": 0,
      "Progress": null
    },
    "source": {
      "Bucket": "minio-go-test-91gscyzoxhpj2lv1",
      "Object": "srcObject",
      "VersionID": "",
      "MatchETag": "",
      "NoMatchETag": "",
      "MatchModifiedSince": "0001-01-01T00:00:00Z",
      "MatchUnmodifiedSince": "0001-01-01T00:00:00Z",
      "MatchRange": false,
      "Start": 0,
      "End": 0,
      "Encryption": null
    }
  },
  "duration": 497,
  "error": "This copy request is illegal because it is trying to copy an object to itself without changing the object's metadata, storage class, website redirect location or encryption attributes.",
  "function": "CopyObject(destination, source)",
  "message": "CopyObject failed",
  "name": "minio-go: testEncryptedSSES3ToSSES3CopyObject",
  "status": "FAIL"
}
(9/15) Running minio-java tests ... done in 21 seconds
(10/15) Running minio-js tests ... done in 36 seconds
(11/15) Running minio-py tests ... done in 1 minutes and 12 seconds
(12/15) Running s3cmd tests ... done in 1 minutes and 0 seconds
(13/15) Running s3select tests ... done in 2 seconds
(14/15) Running security tests ... done in 0 seconds

Executed 14 out of 15 tests successfully.
```
<a name="a960263d-1b36-4e47-bbf3-43022a790173"></a>
### 10285-820793f/mint-zoned.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-c2.minio.io:31405
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 52ad6b49adc7:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 3 seconds
(2/15) Running aws-sdk-java tests ... done in 2 seconds
(3/15) Running aws-sdk-php tests ... done in 44 seconds
(4/15) Running aws-sdk-ruby tests ... done in 5 seconds
(5/15) Running awscli tests ... done in 1 minutes and 18 seconds
(6/15) Running healthcheck tests ... done in 0 seconds
(7/15) Running mc tests ... done in 30 seconds
(8/15) Running minio-dotnet tests ... done in 58 seconds
(9/15) Running minio-go tests ... done in 3 minutes and 6 seconds
(10/15) Running minio-java tests ... done in 1 minutes and 58 seconds
(11/15) Running minio-js tests ... done in 55 seconds
(12/15) Running minio-py tests ... done in 3 minutes and 28 seconds
(13/15) Running s3cmd tests ... FAILED in 23 seconds
{
  "name": "s3cmd",
  "duration": "11542",
  "function": "test_sync_list_objects",
  "status": "FAIL",
  "error": "WARNING: Bucket is not empty. Removing all the objects from it first. This may take some time...\nERROR: S3 error: 404 (Not Found)"
}
(13/15) Running s3select tests ... done in 9 seconds
(14/15) Running security tests ... done in 0 seconds

Executed 14 out of 15 tests successfully.
```
<a name="bd623953-953e-4533-817a-00ad21bf7ffc"></a>
### 10285-820793f/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev7.minio.io:31721
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 0081df392e64:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 8 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 4 minutes and 27 seconds
(4/15) Running aws-sdk-ruby tests ... done in 17 seconds
(5/15) Running awscli tests ... FAILED in 24 seconds
{"name": "awscli", "duration": 1977, "function": "aws --endpoint-url http://minio-dev7.minio.io:31721 s3api put-object --body /mint/data/datafile-0-b --bucket \nAn error occurred (BucketAlreadyOwnedByYou) when calling the CreateBucket operation: Your previous request to create the named bucket succeeded and you already own it. --key datafile-0-b\n", "status": "FAIL", "error": "usage: 
Note: AWS CLI version 2, the latest major version of the AWS CLI, is now stable and recommended for general use. For more information, see the AWS CLI version 2 installation instructions at: https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.htmlusage: aws [options] <command> <subcommand> [<subcommand> ...] [parameters]To see help text, you can run:  aws help  aws <command> help  aws <command> <subcommand> helpUnknown options: occurred, (BucketAlreadyOwnedByYou), when, calling, the, CreateBucket, operation:, Your, previous, request, to, create, the, named, bucket, succeeded, and, you, already, own, it., error"}
(5/15) Running healthcheck tests ... done in 0 seconds
(6/15) Running mc tests ... done in 3 minutes and 47 seconds
(7/15) Running minio-dotnet tests ... done in 1 minutes and 31 seconds
(8/15) Running minio-go tests ... FAILED in 5 minutes and 24 seconds
{
  "args": {},
  "duration": 946,
  "error": "The specified multipart upload does not exist. The upload ID may be invalid, or the upload may have been aborted or completed.",
  "function": "CopyObjectPart(destination, source)",
  "message": "CompleteMultipartUpload call failed",
  "name": "minio-go: testUnencryptedToSSES3CopyObjectPart",
  "status": "FAIL"
}
(8/15) Running minio-java tests ... FAILED in 4 seconds
{
  "name": "minio-java",
  "function": "putObject()",
  "args": "[multi-part upload]",
  "duration": 654,
  "status": "FAIL",
  "error": "java.net.SocketException: Connection reset by peer (Write failed) >>> [java.base/java.net.SocketOutputStream.socketWrite0(Native Method), java.base/java.net.SocketOutputStream.socketWrite(SocketOutputStream.java:110), java.base/java.net.SocketOutputStream.write(SocketOutputStream.java:150), okio.Okio$1.write(Okio.java:79), okio.AsyncTimeout$1.write(AsyncTimeout.java:180), okio.RealBufferedSink.emitCompleteSegments(RealBufferedSink.java:179), okio.RealBufferedSink.write(RealBufferedSink.java:42), okhttp3.internal.http1.Http1ExchangeCodec$KnownLengthSink.write(Http1ExchangeCodec.java:324), okio.ForwardingSink.write(ForwardingSink.java:35), okhttp3.internal.connection.Exchange$RequestBodySink.write(Exchange.java:231), okio.RealBufferedSink.emitCompleteSegments(RealBufferedSink.java:179), okio.RealBufferedSink.write(RealBufferedSink.java:117), io.minio.HttpRequestBody.writeTo(HttpRequestBody.java:78), okhttp3.internal.http.CallServerInterceptor.intercept(CallServerInterceptor.java:69), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142), okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:43), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117), okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:94), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117), okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142), okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:88), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117), okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:229), okhttp3.RealCall.execute(RealCall.java:81), io.minio.MinioClient.execute(MinioClient.java:1014), io.minio.MinioClient.uploadPart(MinioClient.java:7781), io.minio.MinioClient.putObject(MinioClient.java:4706), io.minio.MinioClient.putObject(MinioClient.java:4881), FunctionalTest.testPutObject(FunctionalTest.java:740), FunctionalTest.putObject_test(FunctionalTest.java:800), FunctionalTest.runTests(FunctionalTest.java:3789), FunctionalTest.main(FunctionalTest.java:4056)]"
}
(8/15) Running minio-js tests ... done in 2 minutes and 37 seconds
(9/15) Running minio-py tests ... done in 10 minutes and 20 seconds
(10/15) Running s3cmd tests ... done in 1 minutes and 27 seconds
(11/15) Running s3select tests ... done in 1 minutes and 4 seconds
(12/15) Running security tests ... done in 0 seconds

Executed 12 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Pull request merged**